RPIPs/RPIP-31.md

```diff
@@ -28,18 +28,45 @@ https://dao.rocketpool.net/t/withdrawal-address-splitter-contract/563
 
 ## Specification
 - The withdrawal address that has existed to date SHALL be known as the primary withdrawal address
+
+### Defaults
+- The RPL withdrawal address SHALL be initialized as unset for new node operators
+- The RPL withdrawal address SHALL be initialized as unset for existing node operators
+  
+### Rewards & Withdrawn RPL
 - If a node's RPL withdrawal address is set, all RPL rewards and withdrawn RPL will go to the RPL withdrawal address
 - If a node's RPL withdrawal address is unset, all RPL rewards and withdrawn RPL will go to the primary withdrawal address
+
+### Setting the Address
 - As a node operator, I MUST be able to set an RPL withdrawal address so that I can direct my RPL bond/rewards
   - If a node's RPL withdrawal address is unset, the change MUST come from the node's primary withdrawal address
   - If a node's RPL withdrawal address is set, the change MUST come from the current RPL withdrawal address
 - As a node operator, I MUST be able to confirm the RPL withdrawal address from the new target address before it takes effect, to ensure it is the correct address
 - As the owner of the RPL withdrawal address, I MUST be able to unset my RPL withdrawal address
+
+### Withdrawing Excess RPL
 - As the controller of the RPL withdrawn from a node, I MUST be able to call `RocketNodeStaking.withdrawRPL()` to withdraw excess RPL
   - If a node's RPL withdrawal address is unset, the call MUST come from one of: the node's primary withdrawal address, or the node's address
   - If a node's RPL withdrawal address is set, the call MUST come from the current RPL withdrawal address
-- The RPL withdrawal address SHALL be initialized as unset for new node operators
-- The RPL withdrawal address SHALL be initialized as unset for existing node operators
+
+### Claiming RPL rewards
```

Comment:
Might make sense to have a "Claiming ETH rewards" section for completion here?

Reply:
hmm not sure what we would put in that section - are there relevant items to be addressed for ETH claiming?

Reply:
Yes! The thing I was missing from above!

Namely that the RPL investor can force an ETH distribution if in smoothing pool